src/realTimeReporter/utils.ts

```diff
@@ -9,3 +9,53 @@ export const subscribeToEvent = (event: EVENTS, callback: (params: any) => void)
   // @ts-ignore
   process.on(event, callback);
 };
+
+export const getCodeRef = (itemName: string): string => {
+  const caller = getCaller();
+  const testPath = caller.getFileName();
+  const workingDir = process.cwd();
+
+  const codeRef = testPath
+      .replace(`${workingDir}\\`, '')
```

Comment:
Looks like doesn't work on POSIX systems because of different delimiter:
<img width="564" alt="изображение" src="https://user-images.githubusercontent.com/2759038/77523046-374a7e00-6e96-11ea-9941-7b376e639961.png">

Please see the docs: https://nodejs.org/dist/latest-v12.x/docs/api/path.html#path_path_sep

Reply:
Thought about that. Thank you!